deploy/src/main/resources/conf/app.conf

```diff
@@ -6,7 +6,7 @@
 
 # If mode is DEVELOPMENT or STAGING, UI provides some extra tools
 environment = STAGING
-site = GS
+site = ${?SITE}
```

Comment:
I think that if you use `${SITE}` (without the `?`) the app will fail in the absence of the env variable. Maybe that's what we want.

Reply:
Right we do want that